### PR TITLE
Package sodium-fmt.1.0.0

### PR DESCRIPTION
**Pull request by schubev**
### `sodium-fmt.1.0.0`
Fmt formatters for Sodium
sodium-fmt provides a set of formatters to format values of the various types defined by sodium. It also provides utility formatters which will read from bytes and Sodium.bigbytes values and format them as sodium values.

---
* Homepage: https://git.sr.ht/~schube/sodium-fmt
* Source repo: git+ssh://git@git.sr.ht/~schube/sodium-fmt
* Bug tracker: https://lists.sr.ht/~schube/sodium-fmt

---
:camel: Pull-request generated by opam-publish v2.1.0

### CODE REVIEW

File: packages/sodium-fmt/sodium-fmt.1.0.0/opam [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+synopsis: "Fmt formatters for Sodium"
+description:
+  "sodium-fmt provides a set of formatters to format values of the various types defined by sodium. It also provides utility formatters which will read from bytes and Sodium.bigbytes values and format them as sodium values."
+maintainer: ["~schube/sodium-fmt@lists.sr.ht"]
+authors: ["Victor Schubert"]
+license: "GPL-3.0-or-later"
+depends: [
+  "alcotest" {with-test}
+  "ocaml" {>= "4.13.1"}
+  "dune" {>= "3.0" & >= "3.0.3"}
```
Comment (kit-ty-kate) [inline packages/sodium-fmt/sodium-fmt.1.0.0/opam:12]:
```suggestion
  "dune" {>= "3.0.3"}
```

File: packages/sodium-fmt/sodium-fmt.1.0.0/opam [hunk 2]
```diff
@@ -0,0 +1,42 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "Fmt formatters for Sodium"
+description:
+  "sodium-fmt provides a set of formatters to format values of the various types defined by sodium. It also provides utility formatters which will read from bytes and Sodium.bigbytes values and format them as sodium values."
```
Comment (kit-ty-kate) [inline packages/sodium-fmt/sodium-fmt.1.0.0/opam:5]:
```suggestion
description: """
sodium-fmt provides a set of formatters to format values of the various types defined by sodium.
It also provides utility formatters which will read from bytes and Sodium.bigbytes values and format them as sodium values.
"""
```

File: packages/sodium-fmt/sodium-fmt.1.0.0/opam [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    jobs
+    "@install"
+    "@runtest" {with-test}
+    "@doc" {with-doc}
+  ]
+]
+homepage: "https://git.sr.ht/~schube/sodium-fmt"
+bug-reports: "https://lists.sr.ht/~schube/sodium-fmt"
+doc: "https://schube.srht.site/sodium-fmt/index.html"
+dev-repo: "git+ssh://git@git.sr.ht/~schube/sodium-fmt"
```
Comment (kit-ty-kate) [inline packages/sodium-fmt/sodium-fmt.1.0.0/opam:35]: This repository is not accessible through ssh without write access. https should be used instead
```suggestion
dev-repo: "git+https://git.sr.ht/~schube/sodium-fmt"
```

**Comment by schubev** [comment]
Thanks for the fixes!

**Comment by kit-ty-kate** [comment]
Thanks! Could you return the fixes to the opam file upstream?

**Comment by schubev** [comment]
I have returned the change to the Git URL upstream. I’m unsure however what to do with the description formatting and redundant dune version constraints: this is how dune generated this opam file so any change I make to those fields will be overwritten by dune. I can’t find anything on this; would you happen to know if this is a known issue with a workaround?

**Comment by kit-ty-kate** [comment]
you can use `"""` in the dune-project file as well, this will keep the formatting. But in general keeping the opam files generated by dune is not really necessary all that much so you could also disable it

**Pull request merged**